fuel-compression/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "fuel-compression"
+version = { workspace = true }
+authors = { workspace = true }
+categories = ["cryptography::cryptocurrencies"]
+edition = { workspace = true }
+homepage = { workspace = true }
+keywords = ["blockchain", "cryptocurrencies", "fuel-compression"]
+license = { workspace = true }
+repository = { workspace = true }
+description = "Compression and decompression of Fuel blocks for DA storage."
+
+[dependencies]
+anyhow = { version = "1.0" }
+fuel-derive = { workspace = true }
+paste = "1.0"
+postcard = { version = "1.0", features = ["use-std"], optional = true }
+serde = { version = "1.0", features = ["derive"] }
+serde-big-array = "0.5"
+
+[dev-dependencies]
+fuel-compression = { workspace = true, features = ["test-helpers"] }
```

Comment:
These self-dependencies to enable features are again causing issues. This time it's the publish-crates-check, which requires the crate to be published to be able to do this.